challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |word| word.capitalize }
```

Comment:
There's a shorthand for this: [Symbol#to_proc](http://ruby-doc.org/core-2.0.0/Symbol.html#method-i-to_proc)